mavis/ExtensionManager.hpp

```diff
@@ -1628,6 +1842,21 @@ namespace mavis::extension_manager
             refresh_();
         }
 
+        // Returns whether this extension is defined in Mavis
+        bool isExtensionSupported(const std::string & ext)
+        {
+            assertISAInitialized_();
+            return enabled_arch_->second.isSupported(ext);
+        }
+
```

Comment:
@kathlenemagnus you can use this method to check if an extension is defined in the JSONs.